<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidgetClass</class>
 <widget class="QWidget" name="LoginWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/**/
QWidget#loginPage_, loginPage_2_, registerPage_, settingPage_ {
	background-image: url(:/res/background/loginBackground.jpg);
}
QWidget#settingPage_ {
	background-image: url(:/res/background/loginBackground.jpg);
}


/*  loginPage2_  */
QWidget#loginPage2_ {
	background-image: url(:/res/background/loginBackground.jpg);
}

QLineEdit{
    border: 1px solid #ccc; /* 边框样式，可根据需要调整 */
    border-radius: 10px;    /* 设置边框的圆角半径 */
}

QPushButton#LoginBtn_ , LoginBtn2_ { 
	background-color: rgb(0, 153, 255);
    border: 1px solid #ccc; /* 边框样式，可根据需要调整 */
    border-radius: 10px;    /* 设置边框的圆角半径 */
}
QComboBox {
    border: 1px solid #ccc; /* 边框样式，可根据需要调整 */
    border-radius: 10px;    /* 设置边框的圆角半径 */
}


/*QCombobox主体*/
QComboBox {
    border: 2px solid #f3f3f3;/*设置线宽*/
	/*background-color: rgb(237, 242, 255);背景颜色*/
    /*border-radius: 15px;圆角*/
    padding: 1px 2px 1px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 9em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	/*border-width:2px;边框宽度*/
	/*border-color:rgb(77, 123, 255);边框颜色*/

	padding-left: 10px;/*左侧边距*/
}
/*QCombobox右侧按钮*/
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;/*放于右方顶部*/
    width: 50px;/*设置按钮范围宽度*/
 	/*border-radius: 15px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;*/

    border-top-right-radius: 3px;/*设置边框圆角*/
    border-bottom-right-radius: 3px;
/*padding-right: 50px;*/
}
/*QCombobox右侧按钮的箭头图标*/
QComboBox::down-arrow {
	border-image: url(:/res/icon/xiala.png);/*自定义图片填充*/
	width: 10px;/*设置该图标的宽高*/
	height: 10px;
}


/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
    border: 2px solid #f3f3f3;/*边框宽度、线形、颜色*/
	background-color: rgba(237, 242, 255, 1);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 1px 2px 1px 2px;  /*针对于组合框中的文本内容*/
    min-width: 9em;   /*# 组合框的最小宽度*/
}

/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
	border-radius: 15px;/*圆角*/
    height: 30px;   /* 项的高度（设置pComboBox-&gt;setView(new QListView());后，该项才起作用） */
	background-color: rgb(237, 242, 255);

}

/*以下部分不知为何不生效，有待调试*/
/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover {
    color: #FFFFF0;
       /* 整个下拉窗体越过每项的背景色 */
	background-color: rgb(98, 0, 255);
}

/* 下拉后，整个下拉窗体被选择的每项的样式 */
QComboBox QAbstractItemView::item:selected {
    color: #FFFFF0;
	background-color: rgb(0, 85, 200);
}


</string>
  </property>
  <widget class="QStackedWidget" name="loginPages_">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>400</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="loginPage_">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="RoundLabel" name="headLab_">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../IMClient.qrc">:/res/head/head1.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="SettingBtn_">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../IMClient.qrc">
       <normaloff>:/res/icon/pifu.png</normaloff>:/res/icon/pifu.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="CloseBtn_">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../IMClient.qrc">
       <normaloff>:/res/icon/close.png</normaloff>:/res/icon/close.png</iconset>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="UserId_">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>320</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    border: 1px solid #ccc; /* 边框样式，可根据需要调整 */
    border-radius: 10px;    /* 设置边框的圆角半径 */
}</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>1291035260</string>
      </property>
      <property name="icon">
       <iconset resource="../IMClient.qrc">
        <normaloff>:/res/icon/Contact.png</normaloff>:/res/icon/Contact.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2024416075</string>
      </property>
      <property name="icon">
       <iconset resource="../IMClient.qrc">
        <normaloff>:/res/icon/Contact.png</normaloff>:/res/icon/Contact.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="PassWord_">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>320</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>044438</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>380</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>已阅读并同意服务协议和IM隐私保护指引</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginBtn_">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>430</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerBtn_">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>520</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(68, 134, 232);</string>
     </property>
     <property name="text">
      <string>注册账号</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="moreBtn_">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>520</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(68, 134, 232);</string>
     </property>
     <property name="text">
      <string>更多选项</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>510</y>
       <width>16</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
      </font>
     </property>
     <property name="text">
      <string>|</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loginPage2_">
    <property name="styleSheet">
     <string notr="true">/*  loginPage2_  */
QPushButton#LoginBtn2_ { 
	background-color: rgb(0, 153, 255);
    border: 1px solid #ccc; /* 边框样式，可根据需要调整 */
    border-radius: 10px;    /* 设置边框的圆角半径 */
}</string>
    </property>
    <widget class="QLabel" name="QQLab_">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../IMClient.qrc">:/res/background/QQ.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="RoundLabel" name="headLab2_">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../IMClient.qrc">:/res/head/head1.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>北大西洋暖流</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginBtn2_">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>410</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="SettingBtn_1">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../IMClient.qrc">
       <normaloff>:/res/icon/pifu.png</normaloff>:/res/icon/pifu.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="CloseBtn_1">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../IMClient.qrc">
       <normaloff>:/res/icon/close.png</normaloff>:/res/icon/close.png</iconset>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>500</y>
       <width>16</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
      </font>
     </property>
     <property name="text">
      <string>|</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="moreBtn_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>510</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(68, 134, 232);</string>
     </property>
     <property name="text">
      <string>更多选项</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="registerBtn_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>510</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(68, 134, 232);</string>
     </property>
     <property name="text">
      <string>注册账号</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="registerPage_">
    <property name="styleSheet">
     <string notr="true">background-image: url(:/res/background/loginBackground.jpg);</string>
    </property>
    <widget class="QPushButton" name="CloseBtn_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../IMClient.qrc">
       <normaloff>:/res/icon/close.png</normaloff>:/res/icon/close.png</iconset>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="SettingBtn_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../IMClient.qrc">
       <normaloff>:/res/icon/pifu.png</normaloff>:/res/icon/pifu.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settingPage_">
    <property name="styleSheet">
     <string notr="true">QPushButton#Certain_ { 
	background-color: rgb(0, 153, 255);
    border: 1px solid #ccc; /* 边框样式，可根据需要调整 */
    border-radius: 6px;    /* 设置边框的圆角半径 */
}
QPushButton#Cancel_ { 
	background-color: rgb(0, 153, 255);
    border: 1px solid #ccc; /* 边框样式，可根据需要调整 */
    border-radius: 6px;    /* 设置边框的圆角半径 */
}</string>
    </property>
    <widget class="QPushButton" name="CloseBtn_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../IMClient.qrc">
       <normaloff>:/res/icon/close.png</normaloff>:/res/icon/close.png</iconset>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="SettingBtn_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../IMClient.qrc">
       <normaloff>:/res/icon/pifu.png</normaloff>:/res/icon/pifu.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>网络设置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>类型</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP地址</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>端口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Certain_">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>500</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Cancel_">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>500</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PortEdit_">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>340</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>9999</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="cursorPosition">
      <number>4</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="AddrEdit_">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Tcp</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RoundLabel</class>
   <extends>QLabel</extends>
   <header>E:\StudyVideo\IMClient\src\IMClient\widget\RoundLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../IMClient.qrc"/>
  <include location="../IMClient.qrc"/>
 </resources>
 <connections/>
</ui>
